plugins/sf2_player/sf2_player.cpp

```diff
@@ -184,6 +184,7 @@ sf2Instrument::sf2Instrument( InstrumentTrack * _instrument_track ) :
 
 	InstrumentPlayHandle * iph = new InstrumentPlayHandle( this, _instrument_track );
 	Engine::mixer()->addPlayHandle( iph );
+
```

Comment:
Could you revert unnecessary whitespace changes(this includes new lines) if you can?

Reply:
Made the fix https://github.com/LMMS/lmms/pull/5084/commits/37e151b5fc1e7661d153a782e449e6db91227257